core/controllers/reader.py

```diff
@@ -1011,3 +1011,34 @@ def post(self, exploration_id):
             self.payload.get('report_text'),
             self.user_id)
         self.render_json(self.values)
+
+
+class QuestionPlayerHandler(base.BaseHandler):
+    """Provides questions with given skill ids."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        """Handles GET request."""
+        start_cursor = self.request.get('start_cursor')
+        skill_ids = self.request.get('skill_ids').split(',')
+        question_count = self.request.get('question_count')
+
+        if not question_count.isdigit() or int(question_count) <= 0:
+            raise self.InvalidInputException("question count has " +
```

Comment:
When doing multiline stuff, break after '(' to avoid having to push everything far to the right. Also, use single quotes in backend.

@oppia/dev-workflow-team -- these should be lint checks?

Also, capitalize "Question" in the error message.

Reply:
Apologies for the delay.
Filed #6495.
Thanks.

Reply:
👍 done